sqlglot/dialects/hive.py

```diff
@@ -466,6 +477,12 @@ class Generator(generator.Generator):
             exp.NumberToStr: rename_func("FORMAT_NUMBER"),
             exp.LastDateOfMonth: rename_func("LAST_DAY"),
             exp.National: lambda self, e: self.national_sql(e, prefix=""),
+            exp.Constraint: constraint_sql,
+            exp.ClusteredColumnConstraint: lambda self, e: f"({self.expressions(e, 'this', indent=False)})",
+            exp.NonClusteredColumnConstraint: lambda self, e: f"({self.expressions(e, 'this', indent=False)})",
+            exp.NotForReplicationColumnConstraint: lambda self, e: "",
```

Comment:
i think you could just change the properties location intsead of doing this

Reply:
change it to UNSUPPORTED location property

Reply:
I'm really not understanding this feedback.

Do you mean the "property's location" or the "location property"?
'location' property as in table location as in the storage path, and how does that relate to a clustered constraint?

Reply:
look up Properties.Location and how it's used, switch it to UNSUPPORTED

Reply:
The pattern appears to be to return "" for unsupported options.